src/excalidraw-backend/server.ts

```diff
@@ -213,7 +218,7 @@ export class Server {
             this.utilService.getUserInfoForRoom(userId, roomId),
         );
 
-        if (socket.data.update) {
+        if (socket.data.collaborator) {
```

Comment:
please, do you mind renaming collaborator to something like `hasUpdatePrivilege` or permission, or `canUpdate` or something like that, took me a little while until I saw the comment

Reply:
I think the name of the variable must be independant of the conditions it needs to meet.
IMO it must represent a role / use-case.